### PR TITLE
[MCC-199437] Cover page and other stuffs

### PR DESCRIPTION
**Pull request by chenghuang-mdsol**
Fix bug for Date type
Add cover page from template excel file
Add fixed column width
@art2003 @junshao Please review and merge

### CODE REVIEW

File: Medidata.Rave.Tsdv.Loader/Medidata.Rave.Tsdv.Loader.csproj [hunk 1]
```diff
@@ -89,7 +89,9 @@
   <ItemGroup>
     <None Include="Medidata.Rave.Tsdv.Loader.nuspec" />
     <None Include="packages.config" />
-    <None Include="Resources\CoverSheet.xlsx" />
+    <None Include="Resources\CoverSheet.xlsx">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```
Comment (junshao) [inline Medidata.Rave.Tsdv.Loader/Medidata.Rave.Tsdv.Loader.csproj]: Does it need to be copied? It's can be compiled in the the assembly right?

File: Medidata.Rave.Tsdv.Loader/Presentations/Models/BlockPlanSetting.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+namespace Medidata.Rave.Tsdv.Loader.Presentations.Models
+{
+    public class BlockPlanSetting : IBlockPlanSetting
+    {
+        public string BlockPlanName { get; set; }
+        public string Blocks { get; set; }
+        public int BlockSubjectCount { get; set; }
+        public bool Repeated { get; set; }
+    }
+}
```
Comment (junshao) [inline Medidata.Rave.Tsdv.Loader/Presentations/Models/BlockPlanSetting.cs]: I think we removed all models already.

Comment (junshao) [inline Medidata.Rave.Tsdv.Loader/Presentations/Models/BlockPlanSetting.cs]: This file can be removed.

File: Medidata.Rave.Tsdv.Loader/Medidata.Rave.Tsdv.Loader.csproj (hunk 1 above)
Comment (chenghuang-mdsol) [inline Medidata.Rave.Tsdv.Loader/Medidata.Rave.Tsdv.Loader.csproj]: will change

File: Medidata.Rave.Tsdv.Loader/Presentations/Models/BlockPlanSetting.cs (hunk 2 above)
Comment (chenghuang-mdsol) [inline Medidata.Rave.Tsdv.Loader/Presentations/Models/BlockPlanSetting.cs]: some zombie file, will remove

**Comment by chenghuang-mdsol** [comment]
changes reflected
@junshao

**Comment by junshao** [comment]
LGTM. Thanks. Merging.

**Pull request merged**